### DISCUSSION TITLE
WTS / WES both works?

### DISCUSSION THREAD

**Discussion by linuxbioinfoguy** [liulab-dfci/TRUST4#102] [General]
just want to make sure, I assume using WTS data is the best for this package, but correct me if I am wrong, or it can both works and filter out those non-recombinant reads in the corresponding region from cells that are not immunocell-origin ( only ) ? Thanks!

**Comment by mourisl** [comment]
I have tested TRUST4 on WXS sequencing data before, and it worked and reported a decent number of CDR3s. But I haven't thoroughly evaluated those results yet. 

Those could be from the reads that partially matched V, J, C genes so the exon primers can still capture them.
